views/index.html

```diff
@@ -32,6 +32,10 @@
     <title>
       Federalist<% if (siteDisplayEnv) { %> | <%= siteDisplayEnv %><% } %>
     </title>
+
+    <script>
+      window.FRONTEND_CONFIG = <%- JSON.stringify(frontendConfig) %>;
```

Comment:
here's how we get the `frontendConfig` from the server side to the front end app